backport/backport_pr.py

```diff
@@ -1,8 +1,11 @@
 import gidgethub.routing
 
+import random
+
 from . import tasks
 from . import util
 
+EASTER_EGG = "I'm not a witch! I'm not a witch!"
```

Comment:
Shouldn't this be a list?

Reply:
This doesn't need to be a list, but `random.choice` wasn't needed. I fixed it. Thanks!

Reply:
Cool - I wasn't sure if you were planning for multiple messages.